src/dpi/module.cc

```diff
@@ -118,16 +120,22 @@ class HostDevice {
 
 class MemDevice {
  public:
-  void SetRequest(uint8_t opcode, uint64_t addr, uint32_t len);
-  MemResponse ReadData(uint8_t ready);
-  void WriteData(uint64_t value);
+  void  SetRequest(uint8_t rd_req_valid,uint64_t rd_req_addr, uint32_t rd_req_len, uint32_t rd_req_id, uint64_t wr_req_addr, uint32_t wr_req_len, uint8_t wr_req_valid);
```

Comment:
is the clang formatter catching this?

Reply:
If it is not run by TVM make, than it is not. Fixed long lines in a new PR